package.json

```diff
@@ -35,6 +38,13 @@
   },
   "homepage": "https://github.com/dial-once/sdk-cordova#readme",
   "dependencies": {
-    "cordova": "^7.1.0"
+    "cordova": "^6.5.0"
```

Comment:
You had to downgrade to keep retro-compatibility with our clients apps ?

Reply:
@CAMOBAP795 bump

Reply:
@jkernech I can change it back no problem but as I said anyway we test compilation against different version of cordova `8.0.0 7.1.0 6.5.0`, it was changed to 6.5.0 during manual tests

Rolled back, now